src/alda/repl/commands/ReplCommandManager.java

```diff
@@ -0,0 +1,329 @@
+                          new ReplSave(),
+                          new ReplHelp(this)};
+
+    for (ReplCommand c : cmds) {
+      commands.put(c.key(), c);
+    }
+  }
+
+  /**
+   * Returns the ReplCommand that corresponds to this key
+   * @param key the key to key into
+   * @return the repl command key corresponds to.
+   * null if no command corresponds to key
+   */
+  public ReplCommand get(String key) {
+    return commands.getOrDefault(key, null);
+  }
+
+  /**
+   * Gets a collection of all ReplCommands Available
+   * @return A collection of all the commands we know about
+   */
+  public Collection<ReplCommand> values() {
+    return commands.values();
+  }
+
+  //*************************List of Commands***********************************
+  private class ReplPlay implements ReplCommand {
+    @Override
+    public void act(String args, StringBuffer history, AldaServer server, ConsoleReader reader) {
+      // Parse from/to args
+      String[] arguments = args.split("\\s+");
+      String from = "";
+      String to = "";
+
+      if (arguments.length > 0 && !args.equals("")) {
+        for (int i = 0; i < arguments.length; i++) {
+          if (arguments[i].equals("from")) {
+            // process from argument
+            if (++i >= arguments.length) {
+              usage();
+              return;
+            }
+            from = arguments[i];
+          } else if (arguments[i].equals("to")) {
+            // process to argument
+            if (++i >= arguments.length) {
+              usage();
+              return;
+            }
+            to = arguments[i];
+          } else {
+            usage();
+            return;
+          }
+        }
+      }
+
+      try {
+        server.play(history.toString(), "",
+                    from.length() > 0 ? from : null,
+                    to.length() > 0 ? to : null, false);
+      } catch (Throwable e) {
+        server.error(e.getMessage());
+      }
+    }
+    @Override
+    public String docSummary() {
+      return "Plays the current score.";
+    }
+    @Override
+    public String docDetails() {
+      return "Can take optional `from` and `to` arguments, in the form of markers or mm:ss times.\n\n" +
+        "Without arguments, will play the entire score from beginning to end.\n\n" +
+        "Example usage:\n\n" +
+        "  :play\n" +
+        "  :play from 0:05\n" +
+        "  :play to 0:10\n" +
+        "  :play from 0:05 to 0:10\n" +
+        "  :play from guitarIn\n" +
+        "  :play to verse\n" +
+        "  :play from verse to bridge";
+    }
+    @Override
+    public String key() {
+      return "play";
+    }
+  }
+
+  private class ReplSave implements ReplCommand {
+    private String oldSaveFile = null;
+    @Override
+    public void act(String args, StringBuffer history, AldaServer server, ConsoleReader reader) {
+      // Turn ~ into home
+      args = args.replaceFirst("^~",System.getProperty("user.home"));
```

Comment:
Huh, thats definetly pretty weird...

Can you try compiling the following code snippet and see what it prints out for you? (for me, it prints out `/home/jay`)

```
public class Main {
		public static void main(String[] args) {
				System.out.println(System.getProperty("user.home"));
		}
}
```

I'd rather try to get this working everywhere before this is merged :P